UnitsNet/Scripts/UnitDefinitions/MassFlowRate.json

```diff
@@ -0,0 +1,31 @@
+﻿{
+    "Name": "MassFlowRate",
+    "BaseUnit": "KillogramPerSecond",
+    "XmlDoc": "In physics and engineering mass flow rate is the mass of a substance which passes per unit of time. Its unit is kilogram per second in SI units.",
+    "Units": [
+        {
+            "SingularName": "KillogramPerSecond",
+            "PluralName": "KillogramsPerSecond",
+            "FromUnitToBaseFunc": "x",
+            "FromBaseToUnitFunc": "x",
+            "Localization": [
+                {
+                    "Culture": "en-US",
+                    "Abbreviations": ["kg/s"]
+                }
+            ]
+        },        
+        {
+            "SingularName": "NewtonPerSecond",
```

Comment:
Are you sure NewtonPerSecond is a commonly used unit?
Newton is not a mass unit. I also don't see this unit here:

http://www.onlineconversion.com/flow_rate_mass.htm

My recommendation is to convert from newtons to kilograms first, then use kilogram per second as the representation. We already have Force.FromKilogramForce() or similar, not sure if we have the other way around, Mass.FromNewtons(), but should be trivial to add.